Update to v5.4.0.1

rxm7706 opened the pull request:
<!--
Thank you for your pull request.
Below are a few things we ask you kindly to self-check before getting a review. Remove checks that are not relevant.
-->
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

<!--
Please note any issues this fixes using [closing keywords]( https://help.github.com/articles/closing-issues-using-keywords/ ):
-->
closes https://github.com/conda-forge/podman-py-feedstock/pull/31
closes https://github.com/conda-forge/podman-py-feedstock/pull/34
closes https://github.com/conda-forge/podman-py-feedstock/pull/35
closes https://github.com/conda-forge/podman-py-feedstock/pull/36
closes https://github.com/conda-forge/podman-py-feedstock/pull/37

<!--
Please add any other relevant info below:
-->

rxm7706 committed:
Update to v5.4.0.1

rxm7706 commented:
@conda-forge-admin, please rerender

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/meta.yaml```) and found it was in an excellent condition.

I do have some suggestions for making it better though...


For **recipe/meta.yaml**:

 * ℹ️ `noarch: python` recipes should usually follow the syntax in our [documentation](https://conda-forge.org/docs/maintainer/knowledge_base/#noarch-python) for specifying the Python version.
   - For the `run` section of recipe, you should usually use `python >={{ python_min }}` for the `python` entry.
   - For the `test.requires` section of recipe, you should usually use `python {{ python_min }}` for the `python` entry.
   - If the package requires a newer Python version than the currently supported minimum version on `conda-forge`, you can override the `python_min` variable by adding a Jinja2 `set` statement at the top of your recipe (or using an equivalent `context` variable for v1 recipes).


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/14420400065](https://github.com/conda-forge/conda-forge-webservices/actions/runs/14420400065). Examine the logs at this URL for more detail.</sub>

Commit:
MNT: Re-rendered with conda-build 25.3.2, conda-smithy 3.47.1, and conda-forge-pinning 2025.04.12.06.25.26

rxm7706 committed:
fix linter

rxm7706 commented:
@conda-forge-admin, please rerender

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/meta.yaml```) and found it was in an excellent condition.

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-webservice.

I tried to rerender for you, but it looks like there was nothing to do.


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/14420433890](https://github.com/conda-forge/conda-forge-webservices/actions/runs/14420433890). Examine the logs at this URL for more detail.</sub>

rxm7706 committed:
Fix Url

rxm7706 commented:
@conda-forge-admin, please rerender

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-webservice.

I tried to rerender for you, but it looks like there was nothing to do.


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/14420504789](https://github.com/conda-forge/conda-forge-webservices/actions/runs/14420504789). Examine the logs at this URL for more detail.</sub>

rxm7706 committed:
tomli>=1.2.3 for python_version <3.11

rxm7706 commented:
@conda-forge-admin, please rerender

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-webservice.

I tried to rerender for you, but it looks like there was nothing to do.


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/14420564210](https://github.com/conda-forge/conda-forge-webservices/actions/runs/14420564210). Examine the logs at this URL for more detail.</sub>

rxm7706 committed:
Fix Deps

rxm7706 commented:
@conda-forge-admin, please rerender

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-webservice.

I tried to rerender for you, but it looks like there was nothing to do.


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/14420644504](https://github.com/conda-forge/conda-forge-webservices/actions/runs/14420644504). Examine the logs at this URL for more detail.</sub>

rxm7706 commented:
@4383 @h-vetinari  @alippai 
Could you please take a look and see if this works to update as version 5.4.0.1 
Converted to noarch, updated deps and refreshed the recipe 
Thank you and Regards

h-vetinari commented on recipe/meta.yaml:
```diff
@@ -1,40 +1,47 @@
 source:
-  url: https://github.com/containers/podman-py/archive/refs/tags/v{{ version }}.tar.gz
-  sha256: 464631a8005e61a5118c2d408f373a327efb854074b70ef6e24985d90c0b2231
+  url: https://pypi.org/packages/source/{{ name[0] }}/podman/podman-{{ version }}.tar.gz
```
Please remove all templating of the name, and replace it with its actual value.
```suggestion
  url: https://pypi.org/packages/source/p/podman/podman-{{ version }}.tar.gz
```

h-vetinari reviewed (commented):
Thanks, this looks good, with the exception of introducing the name variable.

rxm7706 committed:
Update recipe/meta.yaml

Co-authored-by: h-vetinari <h.vetinari@gmx.com>

h-vetinari commented on recipe/meta.yaml:
```diff
@@ -1,40 +1,47 @@
-{% set version = "4.5.1" %}
+{% set name = "podman-py" %}
```
```suggestion
```

h-vetinari commented on recipe/meta.yaml:
```diff
@@ -1,40 +1,47 @@
 
 package:
-  name: podman-py
+  name: {{ name|lower }}
```
```suggestion
  name: podman-py
```

h-vetinari reviewed (commented):
> Please remove all templating of the name

rxm7706 committed:
Update recipe/meta.yaml

Co-authored-by: h-vetinari <h.vetinari@gmx.com>

rxm7706 committed:
Update recipe/meta.yaml

Co-authored-by: h-vetinari <h.vetinari@gmx.com>

rxm7706 commented:
@conda-forge-admin, please rerender

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-webservice.

I tried to rerender for you, but it looks like there was nothing to do.


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/14423013905](https://github.com/conda-forge/conda-forge-webservices/actions/runs/14423013905). Examine the logs at this URL for more detail.</sub>

h-vetinari commented:
Thanks!
